apps/dispatch-service/src/app/services/sbatch/sbatch.service.ts

```diff
@@ -257,23 +262,23 @@ srun --job-name="Save-outputs-to-S3" \
       . \
       's3://${storageBucket}/${simulationRunS3Path}'
   aws \
-  --endpoint-url ${storageEndpoint} \
-  s3 sync \
-    . \
-    's3://${storageBucket}/${simulationRunS3Path}' \
-    --exclude '*' \
-    --include '*.h5' \
-    --content-type 'application/hdf5'\
-    --acl public-read
+    --endpoint-url ${storageEndpoint} \
```

Comment:
Are these whitespace changes intentional/tested? I remember that this section was very sensitive to spacing, but not exactly what the issue was.

Reply:
Just to make it more readable. Seemed to work fine.